src/Commands/AutoLeftGear.cpp

```diff
@@ -23,17 +24,20 @@ AutoLeftGear::AutoLeftGear() {
 	// Command1 and Command2 will run in parallel.
   AddSequential(new SetAutoShifter(Chassis::kAutoShiftOff));
     AddSequential(new SetShifter(Chassis::kShifterLow));
-    AddSequential(new AutoDrive(-4.2, 4.2));
-    AddSequential(new AutoDrive(-4.2, 1.25));
-    AddSequential(new AutoDrive(-3.6, 3.6));
+    AddSequential(new AutoDrive(-3.762, 3.673));
+    //AddSequential(new AutoTurn(AutoTurn::kTurnRightSide,1.25));
+    //AddSequential(new AutoTurn(AutoTurn::kTurnLeftSide,-2.2/3));
+    AddSequential(new Delay(.25));
+    AddSequential(new AutoDrive(-3.349, 0.856));
+    AddSequential(new Delay(.25));//orig -4.2, 1.25
+    AddSequential(new AutoDrive(-3.630, 3.722 ));
     AddSequential(new SetSqueeze(GearPickup::kSqueezeOpen));
-    AddParallel(new Delay(1));
     AddSequential(new SetGear(GearPickup::kGearDown));
-    //AddParallel(new GearScore());
+    AddParallel(new Delay(1));
     AddSequential(new SetGearRoller(-RobotMap::kPeakPower));
     AddSequential(new SetGear(GearPickup::kGearDown));
-    AddParallel(new TimedDrive(1,.5,.5));
-    AddSequential(new SetGearRoller(-RobotMap::kPeakPower));
+    //AddParallel(new GearScore());
```

Comment:
Should `GearScore()` be changed to the sequence here? Or should `GearScore()` be deleted here?